application/src/test/java/org/opentripplanner/updater/trip/moduletests/delay/DelayedTest.java

```diff
@@ -74,16 +74,19 @@ void singleStopDelay() {
   void complexDelay() {
     var tripInput = TripInput
       .of(TRIP_2_ID)
-      .addStop(STOP_A1, "0:01:00", "0:01:01")
-      .addStop(STOP_B1, "0:01:10", "0:01:11")
-      .addStop(STOP_C1, "0:01:20", "0:01:21")
+      .addStop(STOP_A1, "0:00:00", "0:00:00")
```

Comment:
Please don't modify existing tests and create new ones instead that test this very feature.